core/src/main/java/org/apache/carbondata/core/metadata/blocklet/BlockletInfo.java

```diff
@@ -221,7 +223,30 @@ public void setNumberOfPages(int numberOfPages) {
       output.writeInt(measureChunksLength.get(i));
     }
     writeChunkInfoForOlderVersions(output);
+    serializeMinMaxValues(output);
+  }
 
+  /**
+   * serialize min max values
+   *
+   * @param output
+   * @throws IOException
+   */
+  private void serializeMinMaxValues(DataOutput output) throws IOException {
```

Comment:
I don't think it is required to serailaize the min/max from driver. if columns are not cached then read footer from executor side.

Reply:
ok..I will remove serialization of min/max and read footer using useMinMaxForPruning flag